src/common/components/footer/Footer.tsx

```diff
@@ -38,6 +49,12 @@ function Footer({ navigationItems }: Props) {
       logoLanguage={logoLanguage}
     >
       <HDSFooter.Navigation variant="minimal">
+        <HDSFooter.Item
+          href="https://www.hel.fi/helsinki/fi/kaupunki-ja-hallinto/osallistu-ja-vaikuta/palaute"
+          label="Anna palautetta"
```

Comment:
- [ ] doesn't this need a translation?

Reply:
- [ ] or Swedish or English link?

Reply:
Oh yes, completely forgot it when other translation stuff came up